### PR TITLE
Let the user pause execution to interactively bring up the host

### PR DESCRIPTION
**Pull request by tiraboschi**
Fixes: https://bugzilla.redhat.com/1686575

### CODE REVIEW

File: tasks/bootstrap_local_vm/05_add_host.yml [hunk 1]
```diff
@@ -99,6 +99,13 @@
       auth: "{{ ovirt_auth }}"
     async: 1
     poll: 0
+  - name: Pause the execution to let the user interactively reconfigure the host
+    block:
+      - name: "Let the user connect to {{ bootstrap_engine_url }} to manually fix host configuration"
```
Comment (irosenzw) [inline tasks/bootstrap_local_vm/05_add_host.yml]: please remove quotation marks

File: tasks/bootstrap_local_vm/05_add_host.yml [hunk 2]
```diff
@@ -99,6 +99,13 @@
...
+        debug:
+          msg: "Continue only when the host is listed as 'up'"
+      - include_tasks: libvirt_default_net_ipv6_tasks.yml
```
Comment (irosenzw) [inline tasks/bootstrap_local_vm/05_add_host.yml]: how this playbook is relevant? did you meant ? :
```
- include_tasks: pause_execution.yml
```

File: tasks/pause_execution.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+- name: Create temporary lock file
+  tempfile:
+    state: file
+    suffix: he_setup_lock
+  delegate_to: localhost
+  register: he_setup_lock_file
+- name: "Pause execution until {{ he_setup_lock_file.path }} is removed, delete it once ready to proceed"
```
Comment (irosenzw) [inline tasks/pause_execution.yml]: please remove quotation marks here also

**Comment by tiraboschi** [comment]
Verified

**Pull request merged**